.gitignore

```diff
@@ -25,3 +25,5 @@ yarn-error.log*
 
 .vscode
 
+yarn.lock
```

Comment:
The lock file is typically _not_ ignored. The lock file tracks exactly which versions of the various modules used in the app were installed, so that any other team member working on a project will have exactly the same versions of those libraries installed. Therefore, we would check in our yarn.lock so that our team members get it when they clone the repo.